src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -163,6 +163,11 @@ public function load(array $configs, ContainerBuilder $container)
             $definition->replaceArgument(1, null);
         }
 
+        $this->addAnnotatedClassesToCache(array(
+            'Bundle\\Controller\\',
+            'Bundle\\Entity\\',
```

Comment:
@stof you missed that this is configured with a chain adapter, that first uses the opcache one (read-only), and fallback to the filesystem (easilly reconfigurable in the same way we can do for any cache pools).